Assets/Scripts/Augment/BulletController.cs

```diff
@@ -34,6 +34,7 @@ protected override void Awake()
     {
         base.Awake();
         UpdateProjectileMovement += ProjectileMotions.MoveWithGravity;
+        bulletsPerShot = bulletsPerShot != 0 ? bulletsPerShot : 1;
```

Comment:
Not sure how this might affect things...

Reply:
I was thinking if we had a barrel where it outputted no projectiles, the knockback effect would still take place as its own extension.

Reply:
Shouldn't this logic be in the extension code then?

Reply:
Yes, on further inspection